appveyor.yml

```diff
@@ -0,0 +1,324 @@
+      MSVCVERSION: v100
+      MSVCYEAR: "vs2010"
+      MSVCABR: "10"
+      VC_VER: 10.0
+      PYTHONPATH: c:\Python34\
+      PYTHONPATHOMNI: "/cygdrive/c/Python34/python"
+      ARCH_PY: win32-msvc10-py34
+      PY_VER: 34
+      BOOST_ROOT: C:/local/boost_1_64_0
+      BOOST_LIBS: C:/local/boost_1_64_0/lib32-msvc-10.0
+      ZMQ_BASE: C:\projects\libzmq
+      IDL_BASE: C:\projects\tangoidl
+      IDL_BIN: C:\Program Files (x86)\tangoidl
+      OMNI_BASE: C:\projects\omniORB-4.2.1
+      BOOST_CFG: >-
+          using python : 3.4 : c:/python34/python.exe : c:/python34/include : c:/python34/libs;
+    - platform: x64
+      ARCH: x64-msvc10
+      configuration: Release
+      CMAKE_GENERATOR: "Visual Studio 10 2010 Win64"
+      MSVCVERSION: v100
+      MSVCYEAR: "vs2010"
+      MSVCABR: "10"
+      VC_VER: 10.0
+      PYTHONPATH: c:\Python34-x64\
+      PYTHONPATHOMNI: "/cygdrive/c/Python34-x64/python"
+      ARCH_PY: x64-msvc10-py34
+      PY_VER: 34
+      BOOST_ROOT: C:/local/boost_1_64_0
+      BOOST_LIBS: C:/local/boost_1_64_0/lib64-msvc-10.0
+      ZMQ_BASE: C:\projects\libzmq
+      IDL_BASE: C:\projects\tangoidl
+      IDL_BIN: C:\Program Files\tangoidl
+      OMNI_BASE: C:\projects\omniORB-4.2.1
+      BOOST_CFG: >-
+          using python : 3.4 : c:/python34-x64/python.exe : c:/python34-x64/include : c:/python34-x64/libs : ^<address-model^>64;
+    - platform: x64
+      ARCH: x64-msvc12
+      configuration: Release
+      CMAKE_GENERATOR: "Visual Studio 12 2013 Win64"
+      MSVCVERSION: v120
+      MSVCYEAR: "vs2013"
+      MSVCABR: "13"
+      VC_VER: 13.0
+      PYTHONPATH: c:\Python34-x64\
+      PYTHONPATHOMNI: "/cygdrive/c/Python34-x64/python"
+      ARCH_PY: x64-msvc12-py34
+      PY_VER: 34
+      BOOST_ROOT: C:/local/boost_1_64_0
+      BOOST_LIBS: C:/local/boost_1_64_0/lib64-msvc-12.0
+      ZMQ_BASE: C:\projects\libzmq
+      IDL_BASE: C:\projects\tangoidl
+      IDL_BIN: C:\Program Files\tangoidl
+      OMNI_BASE: C:\projects\omniORB-4.2.1
+      BOOST_CFG: >-
+          using python : 3.4 : c:/python34-x64/python.exe : c:/python34-x64/include : c:/python34-x64/libs : ^<address-model^>64;
+init:
+  # go to hell Xamarin (see http://help.appveyor.com/discussions/problems/4569)
+  - del "C:\Program Files (x86)\MSBuild\4.0\Microsoft.Common.Targets\ImportAfter\Xamarin.Common.targets"
+  - del "C:\Program Files (x86)\MSBuild\14.0\Microsoft.Common.targets\ImportAfter\Xamarin.Common.targets"
+  - del "C:\Program Files (x86)\MSBuild\12.0\Microsoft.Common.targets\ImportAfter\Xamarin.Common.targets"
+  #RDP from start
+  #- ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+  # Tango
+  - cmd: cd "C:\projects\"
+  - cmd: md libtango
+  - cmd: cd "C:\projects\" 
+  - if %ARCH_PY%==win32-msvc9-py27 appveyor DownloadFile https://ci.appveyor.com/api/buildjobs/fibcw0rnx1ahit1d/artifacts/libtango_9.2.6_v90_x86.zip
```

Comment:
These artefacts are problem (and one of the reasons I said the previous PR would take long to complete).
They don't exist anymore.  AppVeyor only keeps them for a while before deleting.  @NexeyaSGara's idea was to store them in github files linked to releases, e.g. https://github.com/tango-controls/omniorb-windows-ci/releases.  I also see [boost-ci](https://github.com/tango-controls/boost-ci) and [zmq-windows-ci](https://github.com/tango-controls/zmq-windows-ci) repos.
I see we don't have that for https://github.com/tango-controls/cppTango/releases - it just points to AppVeyor.

Besides missing files, we would want to use at least 9.3.3, not 9.2.6.

Reply:
Deferred until appveyor decision is taken (see above)

Reply:
I propse to focus just on windows build on this PR.

Appveyor / Azure devops can come in a future PR.

Reply:
I don't see how you can defer this?  The compilation depends on these artefacts existing.